app/src/main/java/com/github/h3lp3rs/h3lp/professional/ProMainActivity.kt

```diff
@@ -35,4 +37,8 @@ class ProMainActivity : AppCompatActivity() {
         goToActivity(ForumCategoriesActivity::class.java)
     }
 
+    fun goToBasicPortal(view: View) {
```

Comment:
I assume Alexandre already refactored this in his PR, but it would be a good practice in the future to define only 1 "onClick" function and have a switch on the view instead of having 4 goToX

'''
    fun goToActivit(view View) {
    
    when(view) {
    
        ForumCategories -> startActivity(...)
    
'''

Reply:
PS: you don't need to change it I just mention it to make sure we do it right direclty in the future for new activities